src/features/surveys/components/SurveyEditor/index.tsx

```diff
@@ -69,7 +70,21 @@ const SurveyEditor: FC<SurveyEditorProps> = ({ model }) => {
                           handleToggleHidden(elem.id, hidden)
                         }
                       >
-                        <OpenQuestionBlock question={elem.question} />
+                        <OpenQuestionBlock
+                          element={elem.question}
+                          inEditMode={elem.id === idOfBlockInEditMode}
+                          onEditModeEnter={() =>
+                            setIdOfBlockInEditMode(elem.id)
+                          }
+                          onEditModeExit={(
+                            data: Omit<ZetkinTextQuestion, 'required'>
+                          ) => {
```

Comment:
Since this is the same patch body type, I think it should be exported and used here as well.